src/store/undoRedo/index.ts

```diff
@@ -115,15 +124,29 @@ export const actions: ActionTree<IUndoRedoState, IRootState> = {
       flows: rootState.flow,
     })
 
+    let isLastFlowStateFromPersistence = !getters.hasCurrentSnapshot
```

Comment:
this line is unclear 🤔
Could you please clarify?

Reply:
yeah, I agree. 
At the beginning, we don't have currentSnapshot yet, if that's the case, we consider the last flow state was from a persistence action, (eg: after we load a flow)
The original code was
```
let isLastFlowStateFromPersistence = false
if(!getters.hasCurrentSnapshot) {
    isLastFlowStateFromPersistence = true
}
````
Without this, we will have 02 snapshots after the flow load instead of 01.

I think I can leave a comment here to clarify it more

- [ ] leave comment

Reply:
The new code looks more understandable to me, thank you Safidy